specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/Libraries_ListByWorkspace.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-12345678abc",
+    "resourceGroupName": "exampleResourceGroup",
+    "workspaceName": "exampleWorkspace",
+    "ifNoneMatch": null,
+    "api-version": "2019-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 23 Oct 2020 21:33:05 GMT",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-ratelimit-remaining-subscription-reads": "14909",
+        "x-ms-request-id": "f632df92-1140-4e19-9e19-0b1c8cbe393b",
+        "x-ms-correlation-request-id": "f632df92-1140-4e19-9e19-0b1c8cbe393b"
+      },
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.Synapse/workspaces/exampleWorkspaceName/libraryartifacts/exampleLibraryName.jar",
+            "name": "exampleLibraryName.jar",
+            "type": "Microsoft.Synapse/workspaces/libraryartifacts",
+            "properties": {
+              "name": "exampleLibraryName.jar",
+              "path": "exampleWorkspace/libraries/exampleLibraryName.jar",
+              "type": "jar",
+              "uploadedTimestamp": "2020-10-23T07:59:29.1573546+00:00",
+              "provisioningStatus": "Succeeded",
+              "creatorId": "2222222e-222d-222f-2222-22a2f222d2f2"
+            },
+            "etag": "1500474f-0000-0200-0000-5cbe090d0000"
+          }
```

Comment:
Can you add nextLink?
"nextLink": {
          "description": "The link to the next page of results, if any remaining results exist.",
          "type": "string"
        }